events/guildCreate.js

```diff
@@ -2,7 +2,7 @@
 
 module.exports = (client, guild) => {
   // Update the activity to show the updated number of guilds.
-  client.user.setActivity(`${client.settings.get("default").prefix}help | ${client.guilds.cache.size} Servers`);
+  client.user.setActivity(`${client.getSettings("default").prefix}help | ${client.guilds.cache.size} Servers`, {type: "PLAYING"});
```

Comment:
Not sure why you had to add the playing option as status messages default to playing.

Reply:
I didn't knew it was default

Reply:
Yet I tested the original code that does not have `PLAYING` and it defaulted to `PLAYING`, which is why I mentioned it's a needless change.

Unless it changed in 12.5.

[VSCode search](https://i.imgur.com/07jE2XE.png)
[Bot status](https://i.imgur.com/zLwbe5N.png)

Reply:
i just checked the docs and the master branch

Reply:
I stand corrected, we did in fact add it to the master branch, disregard what I said.